docs/api/spec/query-api.yaml

```diff
@@ -0,0 +1,306 @@
+openapi: 3.0.0
+info:
+  title: Treetracker Wallet API
+  contact: {}
+  version: '1.28.0'
+servers:
+  - url: https://localhost:3010
+    variables:
+      environment:
+        default: dev
+        enum:
+          - dev     # Development server
+          - test    # Test server
+paths:
+  '/like/types/{type_id}/things/{object_id}':
+    get:
+      tags:
+        - Like / unlike a likable object
+      summary: Like a likable object
+      description: 'Like a likable object'
+      parameters:
+        - name: type_id
+          description: 'ID of specific token to retrieve'
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+            example: 1
+        - name: object_id
+          description: 'ID of specific token to retrieve'
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+            example: 186734
+      # requestBody:
+      #   description: 'User OIDC access token'
+      #   content:
+      #     text/plain: 
+      #       example: '{{ACCESS_TOKEN}}'
+      #   required: true
```

Comment:
I think we should bring the access token and extract the user id (sub) here right?, so we can record down this in db?

just-count vs save-user-id ?
if we don't save user id for the like operation, how do you avoid repeatedly count the object for a single user?

Reply:
Should we require the client to specify the access token or/and user id in the request body or require the client's access token as part of security header (as in line 248)?